internal/cmd/tunnel.go

```diff
@@ -34,6 +34,10 @@ coder tunnel my-dev 3000 3000
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ctx := context.Background()
 			log := slog.Make(sloghuman.Sink(os.Stderr))
+			if os.Getenv("CODER_TUNNEL_DEBUG") != "" {
```

Comment:
I'm good with this being internal, but I'd prefer if we implemented a `LOG_LEVEL` environment variable in slog longer-term.